rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -277,3 +281,53 @@ def test_find_usd_price_via_second_oracle(inquirer):
     assert price == expected_price
     for oracle_instance in inquirer._oracle_instances[0:2]:
         assert oracle_instance.query_current_price.call_count == 1
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
```

Comment:
Why not mock prices of Aave, Link and CRV here? Or we haven't implemented asset specific mockign in tests yet?